lib/fs.js

```diff
@@ -1144,26 +1143,26 @@ function chmodSync(path, mode) {
 function lchown(path, uid, gid, callback) {
   callback = makeCallback(callback);
   path = getValidatedPath(path);
-  validateUint32(uid, 'uid');
-  validateUint32(gid, 'gid');
+  validateInteger(uid, 'uid', -1, 4294967295);
```

Comment:
Would be better to make `4294967295` a top-level constant (e.g. `const kMaxId = 4294967295;`)